src/Model/Resolver/Cart/Cart.php

```diff
@@ -0,0 +1,93 @@
+<?php
+/**
+ * ScandiPWA - Progressive Web App for Magento
+ *
+ * Copyright © Scandiweb, Inc. All rights reserved.
+ * See LICENSE for license details.
+ *
+ * @license OSL-3.0 (Open Software License ("OSL") v. 3.0)
+ * @package scandipwa/quote-graphql
+ * @link https://github.com/scandipwa/quote-graphql
+ */
+
+declare(strict_types=1);
+
+namespace ScandiPWA\QuoteGraphQl\Model\Resolver\Cart;
+
+use Magento\Framework\GraphQl\Config\Element\Field;
+use Magento\Framework\GraphQl\Exception\GraphQlInputException;
+use Magento\Framework\GraphQl\Schema\Type\ResolveInfo;
+use Magento\Quote\Api\CartManagementInterface;
+use Magento\Quote\Api\GuestCartRepositoryInterface;
+use Magento\QuoteGraphQl\Model\Cart\GetCartForUser;
+use Magento\QuoteGraphQl\Model\Resolver\Cart as SourceCart;
+use Magento\Framework\Exception\NoSuchEntityException;
+use Magento\Webapi\Controller\Rest\ParamOverriderCustomerId;
+use Magento\Quote\Model\QuoteIdToMaskedQuoteIdInterface;
+
+/**
+ * @inheritdoc
+ */
+class Cart extends SourceCart
+{
+    /**
+     * @var ParamOverriderCustomerId
+     */
+    public $overriderCustomerId;
```

Comment:
here and everywhere: please use typed properties
this will be tagged in latest PWA for most part, meaning latest Magento, meaning PHP 7.4 at least 

so
`public ParamOverriderCustomerId $overriderCustomerId;`
etc